packages/flutter/lib/src/widgets/_platform_selectable_region_context_menu_web.dart

```diff
@@ -135,6 +135,9 @@ class PlatformSelectableRegionContextMenu extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Stack(
+      // Setting fit to StackFit.passthrough will prevent the child from misaligning on web
+      // and be consist with other platforms.
+      fit: StackFit.passthrough,
       alignment: Alignment.center,
```

Comment:
I'm not entirely sure about the origins of why we set the alignment to `Alignment.center` here. I tried removing it and didn't notice any glaring issues. cc @chunhtai if you had any context on this.